### ISSUE TITLE
Make installable, run as a service

### ISSUE THREAD

**Issue by ekristen** [JamesHabben/evolve#14]
Can you add a `setup.py` that allows the project to be installed and also run as a service.

**Comment by JamesHabben** [comment]
I think there is one more feature required before this can run as a service. It needs to support choosing the memory dump file through the web GUI. Currently, it is accepted through the parameters, and does some prep work while starting. I have to rearrange some code to have it fire after a memory dump is chosen. That is the next thing on my list. Thanks for the request!

**Comment by ekristen** [comment]
If you can make it installable via pip or Python with a setup.py for now that'd be a good compromise, that way the binary can just end up in usr/local/bin

**Comment by JamesHabben** [comment]
Ah! I combined those into one. I will look into the pip. I have wanted to do this one also.

**Comment by ekristen** [comment]
@JamesHabben Fantastic! 

Ultimately I'd like to see everyone just using this as a docker container, because docker is amazing and is well suited for tasks like this.

I manage the SIFT Workstation for SANS, and we want to add the tool, but I need it to be installed so it is manageable. Thanks!

**Comment by JamesHabben** [comment]
Good to know. Have you seen this issue post?
https://github.com/JamesHabben/evolve/issues/7

**Comment by JamesHabben** [comment]
I just pushed a setup file, but I haven't had a chance to test it yet. Let me know if you do happen to test it.

**Comment by ekristen** [comment]
@JamesHabben installation using `sudo pip install git+https://github.com/JamesHabben/evolve.git` works great. Any chance you can publish it to the python package server?

**Comment by JamesHabben** [comment]
Yup! I created an account, but haven't got it all set properly just yet. Sorry for the delay. I will try to get that all finished very soon.

**Comment by JamesHabben** [comment]
I have version 1.5 published to pip. Let me know if you have any issues with it.

**Comment by ekristen** [comment]
I was wrong. @JamesHabben installing via pip purely does not work. It seems like that setup.py is not setup correctly to tell pip about the evolve.py file or its web resources. 

I'm looking at still adding this to SIFT, but need to be able to install it into a python virtualenv.
